.gitlab/notify/notify.yml

```diff
@@ -26,7 +26,9 @@ notify:
   timeout: 15 minutes # Added to prevent a stuck job blocking the resource_group defined above
   script:
     - export GITLAB_TOKEN=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $GITLAB_READ_API_TOKEN_SSM_NAME)
+    - export DD_API_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $API_KEY_ORG2_SSM_NAME)
     - !reference [.setup_python_mirror_linux]
+    - python3 -m pip install datadog_api_client
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestions:** 
- Could you please put the `datadog_api_client` requirement in a `requirements.txt` file?
- We need to pin `datadog_api_client` as well for reproducibility / security.